src/main/java/site/onandoff/util/encryption/BcryptManager.java

```diff
@@ -0,0 +1,15 @@
+package site.onandoff.util.encryption;
+
+import org.mindrot.jbcrypt.BCrypt;
+
+public class BcryptManager {
+
+	public static String encrypt(String plainPassword) {
+		return BCrypt.hashpw(plainPassword, BCrypt.gensalt());
+	}
+
+	public static boolean isPasswordMatch(String plainPassword, String hashPassword) {
+		return BCrypt.checkpw(plainPassword, hashPassword);
+	}
```

Comment:
💁🏻 BCrypt 카멜 케이스도 통일하는 건 어떤가요?
💁🏻 내부적으로 변수명을 password로 갖고 있어서 '패스워드 암호화 매니저'에 더 가까운 듯 느껴지는데, 독립적으로 생각한다면 다음과 같이 만들 수도 있을 것 같아요
```suggestion
	public static String encrypt(String plainData) {
		return BCrypt.hashpw(plainData, BCrypt.gensalt());
	}

	public static boolean isMatch(String plainData, String hashedData) {
		return BCrypt.checkpw(plainData, hashedData);
	}
```

Reply:
1) BCrypt 카멜 케이스를 통일한다는 게 무슨뜻일까요? 

2) 저희 프로젝트에선 비밀번호에만 암호화를 하니까 저런식으로 네이밍한건데, 말씀하신대로 plainData 의 네이밍을 사용하면 독립적이어서 좋은거 같네요. 
해당 네이밍으로 수정하겠습니다!

Reply:
1. 클래스 이름은 Bcrypt로 되어있어서 말씀드렸어요!
2. 감사합니다 🙇🏻

Reply:
아하 BCrypt 로 바꿀게요~!